src/snowcli/cli/__init__.py

```diff
@@ -152,6 +151,14 @@ def default(
         dir_okay=False,
         is_eager=True,
     ),
+    verbosity: LogVerbosity = typer.Option(
+        LogVerbosity.QUIET.value,
+        "--verbosity",
+        "-v",
+        help="Print more logs",
```

Comment:
Can you provide more information? Should we extend README about how the logging works?

Reply:
I will add more info to command and readme